playwright-e2e/pages/dsm/participantList-page.ts

```diff
@@ -20,6 +24,18 @@ export default class ParticipantListPage {
     await waitForNoSpinner(this.page);
   }
 
+  public async filterListByParticipantGUID(participantGUID: string): Promise<void> {
+    await this.page.locator('text=Customize View >> button').click();
```

Comment:
Instead, you could use customize view functionality, which is available from the `_filters` field. Like so:
```
const customizeViewPanel = await this.filters.customizeViewPanel;
await customizeViewPanel.open();
await customizeViewPanel.selectColumns('Participant Columns', ['Participant ID']);
const searchPanel = await this.filters.searchPanel;
await searchPanel.open();
await searchPanel.text('Participant ID', {textValue:participantGUID});
await searchPanel.search();
```

And later to make assertions.

What do you think?